frontend/src/components/Collection/components/CollectionDatasetsGrid/components/Row/DatsetRow/index.tsx

```diff
@@ -186,9 +212,11 @@ const DatasetRow: FC<Props> = ({
               progress={datasetStatus.upload_progress}
             />
           )}
-          <StatusTags>
-            {revisionsEnabled && <RevisionStatusTag dataset={dataset} />}
-          </StatusTags>
+          {revisionsEnabled && (
```

Comment:
Wrapped `revisionsEnabled` check around the `<StatusTag/>` element to prevent unnecessary render of element.